#!/home/mohama0e/CoT3D_VG/automatic_loc_module/.conda/bin/python3.10
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
